tests/src/test/java/org/jboss/weld/tests/unit/bootstrap/xml/BeansXmlTest.java

```diff
@@ -97,7 +97,7 @@ public void testInterceptorsEnabled() throws Exception {
 
             public void run(WeldManager beanManager) {
                 ModuleEnablement enabled = ((BeanManagerImpl) beanManager).getEnabled();
-                assertEquals(2, enabled.getInterceptors().size());
+                assertEquals(3, enabled.getInterceptors().size());
```

Comment:
Had to change unit tests here - from what I gathered, this tests all enabled interceptors in given module. In this case one interceptor is from test itself, 2 others come from weld core.